definitions/ext-vpc_network/aws-kinesis-firehose-vpc-flow-logs-dashboard.json

```diff
@@ -0,0 +1,195 @@
+{
+  "name": "AWS VPC Flow Logs",
+  "description": null,
+  "pages": [
+    {
+      "name": "AWS VPC Flow Logs",
+      "description": null,
+      "widgets": [
+        {
+          "visualization": {
+            "id": "viz.markdown"
+          },
+          "layout": {
+            "column": 1,
+            "row": 1,
+            "height": 2,
+            "width": 3
+          },
+          "title": "",
+          "rawConfiguration": {
+            "text": "# VPC Flow Log Analytics\n---\n\n![](https://d0.awsstatic.com/logos/powered-by-aws.png)\n\n\n\n"
+          }
+        },
+        {
+          "visualization": {
+            "id": "viz.table"
+          },
+          "layout": {
+            "column": 4,
+            "row": 1,
+            "height": 3,
+            "width": 9
+          },
+          "title": "Detected Anomalies",
+          "rawConfiguration": {
+            "dataFormatters": [],
+            "facet": {
+              "showOtherSeries": false
+            },
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": "FROM WatcherSignalDeviation SELECT facetAttribute AS 'Type', facetValue AS 'IP', status, title AS 'Anomaly', comparisonWindowStandardDeviation AS 'Z-Score', lookoutLink AS 'Explore via Lookout' WHERE status IN('STARTED','ONGOING','RESOLVED')"
```

Comment:
I've been told this will be available soon (~2 weeks). The vpc firehose isn't live yet, so expectation is customers shouldn't see any issues as they won't get this entity/dashboard. We're really just getting an initial dashboard in place for demo purposes.